modules/ROOT/pages/syntax-quick-reference.adoc

```diff
@@ -0,0 +1,1290 @@
+include::lists:example$description.adoc[tag=3-mix]
+====
+
+TIP: Lists can be indented.
+Leading whitespace is not significant.
+
+.Complex content in outline lists
+[#ex-complex]
+----
+include::lists:example$complex.adoc[tag=b-complex]
+----
+
+.View result of <<ex-complex>>
+[%collapsible.result]
+====
+include::lists:example$complex.adoc[tag=b-complex]
+====
+
+== Links
+
+.Autolinks, URL macro, and mailto macro
+[#ex-urls]
+----
+include::macros:example$url.adoc[tag=b-base]
+
+include::macros:example$url.adoc[tag=b-scheme]
+----
+
+.View result of <<ex-urls>>
+[%collapsible.result]
+====
+include::macros:example$url.adoc[tag=b-base]
+
+include::macros:example$url.adoc[tag=b-scheme]
+====
+
+.URL macros with attributes
+[#ex-linkattrs]
+----
+include::macros:example$url.adoc[tag=b-linkattrs]
+----
+
+.View result of <<ex-linkattrs>>
+[%collapsible.result]
+====
+include::macros:example$url.adoc[tag=b-linkattrs]
+====
+
+CAUTION: If the linked text contains a comma and the text is followed by one or more named attributes, you must enclose the text in double quotes.
+Otherwise, the text will be cut off at the comma (and the remaining text will get pulled into the attribute parsing).
+
+.URLs with spaces and special characters
+----
+include::macros:example$url.adoc[tag=b-spaces]
+----
+
+.Link to relative page
+----
+link:index.html[Docs]
+----
+
+.Link using a Windows path
+----
+include::macros:example$url.adoc[tag=b-windows]
+----
+
+.Inline anchors
+----
+include::attributes:example$id.adoc[tag=anchor]
+----
+
+.Cross references
+[#ex-xrefs]
+----
+include::macros:example$xref.adoc[tag=b-base]
+----
+
+.View result of <<ex-xrefs>>
+[%collapsible.result]
+====
+include::macros:example$xref.adoc[tag=b-base]
+====
+
+.Inter-document cross references
+----
+include::macros:example$xref.adoc[tag=b-inter]
+----
+
+== Images
+
+Images are resolved relative to the value of the xref:macros:images-directory.adoc[imagesdir] document attribute, which is empty by default.
+Use the `imagesdir` attribute to avoid hard coding the common path to your images in every image macro.
+The `imagesdir` attribute can be an absolute path, relative path, or base URL.
+When the image target is a URL or absolute path, the imagesdir prefix is _not_ prepended.
+
+.Block image macro
+[#ex-image-blocks]
+----
+include::macros:example$image.adoc[tag=base]
+
+include::macros:example$image.adoc[tag=alt]
+
+include::macros:example$image.adoc[tag=attr]
+
+include::macros:example$image.adoc[tag=ab-url]
+----
+
+.View result of <<ex-image-blocks>>
+[%collapsible.result]
+====
+include::macros:example$image.adoc[tag=qr-base]
+
+include::macros:example$image.adoc[tag=qr-alt]
+
+include::macros:example$image.adoc[tag=qr-attr]
+
+include::macros:example$image.adoc[tag=ab-url]
+====
+
+Two colons following the image keyword in the macro (i.e., `image::`) indicates a block image (aka figure), whereas one colon following the image keyword (i.e., `image:`) indicates an inline image.
```

Comment:
nitpick: I think it should be "a.k.a." for "also known as"

Reply:
Although i.e., and e.g., require the punctuation, aka is accepted as word. At least, that's how I've seen it done in published works. See https://writingexplained.org/aka-or-aka